x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -49,11 +49,11 @@ func (s *KeeperTestSuite) TestMsgCreateDenom() {
 	s.Require().True(preCreateBalance.Sub(postCreateBalance).IsEqual(denomCreationFee[0]))
 
 	// Make sure that a second version of the same denom can't be recreated
-	res, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "bitcoin"))
+	_, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "bitcoin"))
 	s.Require().Error(err)
 
 	// Creating a second denom should work
-	res, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "litecoin"))
+	_, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "litecoin"))
 	s.Require().NoError(err)
 	s.Require().NotEmpty(res.GetNewTokenDenom())
```

Comment:
This doesn't look right. We are using an old `res` variable on line 58 but the intention is to validate the updated one.

Reply:
If a linter is outputting this, we should not use this linter.

Reply:
It was me, I'll take a look.  Thanks for your review. (this was not an automatically generated fix)